akka-docs/src/main/paradox/stream/stream-parallelism.md

```diff
@@ -82,7 +82,7 @@ since that would require a third processing step, which is not practically possi
 
 One drawback of the example code above that it does not preserve the ordering of pancakes. This might be a problem
 if children like to track their "own" pancakes. In those cases the `Balance` and `Merge` operators should be replaced
-by strict-round robing balancing and merging operators that put in and take out pancakes in a strict order.
+by strict-round robining balancing and merging operators that put in and take out pancakes in a strict order.
```

Comment:
Could chuck a comma after `operators` if you want too

Reply:
is "robining" a thing? I would say "by round robin balancing"

"strict" is also mentioned at the end of the sentence so can be removed at the first location